.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
   - '4.3'
   - '4.2'
   - '4.0'
-script: make dist && make test-dist && make lint
+script: make dist && make lint && make test-dist
```

Comment:
I think it should be ` make lint && make dist && make test-dist`, as linting should be the very first task